src/bundle.ts

```diff
@@ -44,15 +38,21 @@ export class Bundler extends Transform {
     this.config = config;
     this.analyzer = analyzer;
     this.sharedBundleUrl = 'shared-bundle.html';
+    this.bundler = new Bundler({
+      analyzer: analyzer.analyzer,
+      inlineCss: true,
+      inlineScripts: true,
+    });
   }
 
   _transform(
       file: File,
       _encoding: string,
       callback: (error?: any, data?: File) => void): void {
-    // If this file is a fragment, hold on to the file so that it's fully
-    // analyzed by the time down-stream transforms see it.
-    if (this.config.isFragment(file.path)) {
+    // If this file is a fragment or an entrypoint, hold on to the file so that
+    // it's fully analyzed by the time down-stream transforms see it.
+    if (this.config.isFragment(file.path) ||
+        !!this.config.entrypoint && this.config.entrypoint === file.path) {
```

Comment:
- `this.config.entrypoint` is always set
- Do we always want to bundle the entrypoint? We currently use this logic for deciding when the entrypoint is a fragment, can you confirm? https://github.com/Polymer/polymer-project-config/blob/master/src/index.ts#L225